templates/orbital-sync.xml

```diff
@@ -0,0 +1,59 @@
+    <Config Name="Sync regex whitelist" Target="SYNC_REGEX_WHITELIST" Default="true" Mode="" Description="Copies the regex whitelist" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
+    <Config Name="Sync blacklist" Target="SYNC_BLACKLIST" Default="true" Mode="" Description="Copies the blacklist" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
+    <Config Name="Sync regexlist" Target="SYNC_REGEXLIST" Default="true" Mode="" Description="Copies the regex blacklist" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
+    <Config Name="Sync adlist" Target="SYNC_ADLIST" Default="true" Mode="" Description="Copies the adlist" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
+    <Config Name="Sync client" Target="SYNC_CLIENT" Default="true" Mode="" Description="Copies clients" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
+    <Config Name="Sync group" Target="SYNC_GROUP" Default="true" Mode="" Description="Copies groups" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
+    <Config Name="Sync audit log" Target="SYNC_AUDITLOG" Default="false" Mode="" Description="Copies the audit log" Type="Variable" Display="advanced" Required="false" Mask="false">false</Config>
+    <Config Name="Sync static dhcp leases" Target="SYNC_STATICDHCPLEASES" Default="false" Mode="" Description="Copies static dhcp leases" Type="Variable" Display="advanced" Required="false" Mask="false">false</Config>
+    <Config Name="Sync local dns records" Target="SYNC_LOCALDNSRECORDS" Default="true" Mode="" Description="Copies local DNS records" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
+    <Config Name="Sync local cname records" Target="SYNC_LOCALCNAMERECORDS" Default="true" Mode="" Description="Copies local CNAME records" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
+    <Config Name="Sync flush tables" Target="SYNC_FLUSHTABLES" Default="true" Mode="" Description="Clears existing data on the secondary (copy target) Pi-hole" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
+    <Config Name="Notify on success" Target="NOTIFY_ON_SUCCESS" Default="false" Mode="" Description="Send a notification if a sync completes successfully." Type="Variable" Display="advanced" Required="false" Mask="false">false</Config>
+    <Config Name="Notify on failure" Target="NOTIFY_ON_FAILURE" Default="true" Mode="" Description="Send notifications if a sync fails for any reason." Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
+    <Config Name="Notify via SMTP" Target="NOTIFY_VIA_SMTP" Default="false" Mode="" Description="Send notifications via email." Type="Variable" Display="advanced" Required="false" Mask="false">false</Config>
```

Comment:
These bools can be presented as a drop-down, as done in [vaultwarden](https://github.com/selfhosters/unRAID-CA-templates/blob/98fb52fa952be8b93da94f1e2ac4658a661090cd/templates/vaultwarden.xml#L47) for the Default value. the true default is the first value in the pipe(|) separated list.